pkg/operator/ceph/object/user.go

```diff
@@ -54,8 +58,16 @@ func decodeUser(data string) (*ObjectUser, int, error) {
 		return nil, RGWErrorParse, errors.Wrapf(err, "failed to unmarshal json. %s", data)
 	}
 
-	rookUser := ObjectUser{UserID: user.ID, DisplayName: &user.DisplayName, Email: &user.Email}
+	rookUser := ObjectUser{UserID: user.ID, DisplayName: &user.DisplayName, Email: &user.Email, MaxBuckets: *user.MaxBuckets}
 
+	if len(user.Caps) > 0 {
+		rookUser.Caps = user.Caps
+	}
+
+	if *user.UserQuota.Enabled {
```

Comment:
Need to check for nil here and for the MaxSize and MaxObjects? Otherwise, seems like the operator could crash

Reply:
Usually by RGW fills out the default value JSON field user.UserQuota , the user.BucketQuota, but it is not harmful to check as weel